group_vars/all.yml.dist

```diff
@@ -181,6 +182,15 @@ samba_shares:
     browsable: yes
     path: "{{ samba_shares_root }}/photos"
 
+###
```

Comment:
Please can you move this whole section down to the "you shouldn't need to edit..." section, then document as per Miniflux

Reply:
Done.